scripts/renumber_corpus_files.py

```diff
@@ -18,9 +19,11 @@
 
 def parse_arguments():
     parser = ArgumentParser(description=__doc__)
-    parser.add_argument('input_dirs', nargs='+',
-                        help='the input directories.')
-    parser.add_argument('--output-dir', '-o', required=True,
+    parser.add_argument('--input-dir', '-i', type=Path, required=True,
```

Comment:
Why? Joining multiple input directories is quite useful.

Reply:
First, because I wanted it to match the generic pattern of args we use in these scripts.

Second: I don't think that supporting the case of multiple input dirs and a single output dir would is that useful.

Reply:
It is, if e.g. I want to put all of `cc`, `cc_ro_hu`, `cc_sk_hu`, etc. into a single directory. I think this is a perfectly valid use-case.

You could just change the parameter to `action='append'`, and throw a `parser.error` if it is empty. Then it can be used in a pipeline in the same way as the rest of the scripts.

Reply:
Okey-dokey!